### ISSUE TITLE
Inheritance class WebAppServer from web.Application is discouraged

### ISSUE THREAD

**Issue by banesullivan** [Kitware/wslink#143]
Seeing this deprecation warning from aiohttp through wslink/trame

```
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/wslink/backends/aiohttp/__init__.py", line 38, in <module>
    class WebAppServer(AbstractWebApp, aiohttp_web.Application):
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/abc.py", line 106, in __new__
    cls = super().__new__(mcls, name, bases, namespace, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/aiohttp/web_app.py", line 161, in __init_subclass__
    warnings.warn(
DeprecationWarning: Inheritance class WebAppServer from web.Application is discouraged
```

**Comment by jourdain** [comment]
I'm guessing that is not with just 3.10.

I should fix that which is pretty easy thankfully.

**Comment by jourdain** [comment]
Fixed in 1.11.1
